cloud/ovirt/ovirt_hosts.py

```diff
@@ -0,0 +1,322 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2016 Red Hat, Inc.
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+try:
+    import ovirtsdk4 as sdk
+    import ovirtsdk4.types as otypes
+
+    from ovirtsdk4.types import HostStatus as hoststate
+except ImportError:
+    pass
+
+from ansible.module_utils.ovirt import *
+
+
+DOCUMENTATION = '''
+---
+module: ovirt_hosts
+short_description: Module to manage hosts in oVirt
+version_added: "2.3"
+author: "Ondra Machacek (@machacekondra)"
+description:
+    - "Module to manage hosts in oVirt"
+options:
+    name:
+        description:
+            - "Name of the the host to manage."
+        required: true
+    state:
+        description:
+            - "Should the host be present/absent/maintenance/upgraded"
+        choices: ['present', 'absent', 'maintenance', 'upgraded', 'started', 'restarted', 'stopped']
+        default: present
+    comment:
+        description:
+            - "Description of the host."
+    cluster:
+        description:
+            - "Name of the cluster, where host should be created."
+    address:
+        description:
+            - "Host address. Can be IP address or FQDN."
```

Comment:
I'd rephrase to: 'Host address. It can be either FQDN (preferred) or IP address.'